### PR TITLE
react init

### PR DESCRIPTION
**Pull request by Nessunskill**
reactjs example

### CODE REVIEW

File: examples/reactjs/package.json [hunk 1]
```diff
@@ -0,0 +1,49 @@
+{
+  "name": "reactjs",
```
Comment (maxval1) [inline examples/reactjs/package.json:2]: rename to `@brizy/reactjs`

File: examples/reactjs/package.json [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@brizy/docs-react": "*",
+    "@brizy/react": "*",
+    "@types/node": "^16.18.38",
+    "@types/react": "^18.2.14",
+    "@types/react-dom": "^18.2.6",
```
Comment (maxval1) [inline examples/reactjs/package.json:10]: normalize all `dependencies` some of the is part of `devDependencies`

File: examples/reactjs/package.json [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    "production": [
+      ">0.2%",
+      "not dead",
+      "not op_mini all"
+    ],
+    "development": [
+      "last 1 chrome version",
+      "last 1 firefox version",
+      "last 1 safari version"
+    ]
```
Comment (maxval1) [inline examples/reactjs/package.json:47]: same `browserslist` for all env

File: examples/reactjs/public/logo192.png
Comment (maxval1) [inline examples/reactjs/public/logo192.png:1]: is used?

File: examples/reactjs/public/logo512.png
Comment (maxval1) [inline examples/reactjs/public/logo512.png:1]: is used?

File: examples/reactjs/src/pages/page.tsx [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+const Page = (): ReactElement => {
+  const location = useLocation();
+  const path = location.pathname.substring(1);
+
+  const [page, setPage] = useState<CompilerData>();
+  const [error, setError] = useState<Error | null>(null);
+
+  const useCache = (cache: string) => {
+    console.log("Found page in cache");
```
Comment (maxval1) [inline examples/reactjs/src/pages/page.tsx:14]: 1. remove `console.log`
2. remove cache

File: packages/sdk/src/client/resource.ts [hunk 5]
```diff
@@ -6,7 +6,9 @@ export class Resource {
 
   constructor(config: ConfigType) {
     this.apiKey = config.apiKey;
-    this.domain = config.devServer ? "demo server" : "https://brizy.cloud/api";
+    this.domain = config.devServer
+      ? "demo server"
```
Comment (maxval1) [inline packages/sdk/src/client/resource.ts:10]: change `demo server` to beta1

File: packages/react/src/Scripts/index.tsx [hunk 6]
```diff
@@ -1,26 +1,73 @@
...
-            dangerouslySetInnerHTML={{ __html: s.html }}
-          />
-        ) : (
-          <script key={i} {...s.attr}></script>
-        )
-      )}
-    </>
+    script.html ===
+    // eslint-disable-next-line
+    `document.addEventListener('DOMContentLoaded', () => window.Brz.emit("init.dom", jQuery(document.body)))`
```
Comment (maxval1) [inline packages/react/src/Scripts/index.tsx:13]: this is main script

File: packages/react/src/Scripts/index.tsx [hunk 7]
```diff
@@ -1,26 +1,73 @@
...
+        });
+      }
+
+      return Promise.resolve();
+    };
+
+    const loadScriptsSequentially = async (scripts: Script[]) => {
+      for (const script of scripts) {
+        await executeScript(script);
+      }
```
Comment (maxval1) [inline packages/react/src/Scripts/index.tsx:46]: need to execute all of this only if is in browser and not SSR

**Review by maxval1 — CHANGES_REQUESTED** [review]

File: packages/react/src/Scripts/index.tsx [hunk 8]
```diff
@@ -1,26 +1,73 @@
...
+    `document.addEventListener('DOMContentLoaded', () => window.Brz.emit("init.dom", jQuery(document.body)))`
   );
 };
+
+const LoadScripts: FunctionComponent<LoadScriptsProps> = ({ scripts }) => {
+  useEffect(() => {
+    const executeScript = (script: Script): Promise<void> => {
+      const scriptElement = document.createElement("script");
+
+      if (script.html && !isBrizyInitScript(script)) {
```
Comment (maxval1) [inline packages/react/src/Scripts/index.tsx:22]: why it's neede?

**Review by maxval1 — CHANGES_REQUESTED** [review]

File: packages/react/src/Scripts/index.tsx (hunk 8 above)
Comment (Nessunskill) [reply packages/react/src/Scripts/index.tsx:22]: If inline script received and it's not brizy main script we should load it. Only main script should be skipped

File: packages/react/src/types.ts [hunk 9]
```diff
@@ -7,3 +7,18 @@ export interface CompilerData {
...
+}
+
+export interface Script {
+  html?: string;
+  attr?: ScriptAttributes;
+}
+
+export interface LoadScriptsProps {
+  scripts: Script[];
+}
```
Comment (maxval1) [inline packages/react/src/types.ts:24]: omit all of them

**Review by maxval1 — CHANGES_REQUESTED** [review]

File: packages/react/src/hooks/useLoadScript.ts [hunk 10]
```diff
@@ -0,0 +1,85 @@
...
+  useEffect(() => {
+    const isServer = window.isServer;
+
+    if (isServer) return;
+
+    loadScriptsSequentially(scripts)
+      .then(() => {
+        try {
+          // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+          // @ts-ignore
```
Comment (maxval1) [inline packages/react/src/hooks/useLoadScript.ts:68]: change `@ts-ignore` to `ts-expect-error: My Message`

**Review by maxval1 — CHANGES_REQUESTED** [review]
Need to add `npm run build:reactjs` in package.json inside root to build only react example same like npm run build:nextjs

**Pull request merged**